Process.h

```diff
@@ -106,7 +114,7 @@ typedef struct ProcessFieldData_ {
    const char* name;
    const char* title;
    const char* description;
-   int flags;
+   uint_least32_t flags;
```

Comment:
i think @natoscott said he does not like fancy integers?

Reply:
Using `int` in this case would be wrong for platforms with `sizeof(int)==sizeof(short)` as this patchset introduces a 17th flag, thus we need at least 32 bit here.

Reply:
> Using `int` in this case would be wrong for platforms with `sizeof(int)==sizeof(short)` [...]

What platform (that htop runs on) has that property?

( +1 to using uint32_t anyway )